File: src/data/snippets/while-loops/snippet.cs
// Initialize a counter
int count = 10;  // Starting countdown number

while (count >= 0)
{
    // Skip even numbers
    if (count % 2 == 0)
    {
        count--;
        continue;
    }

    // Print the current count
    Console.WriteLine(count);
    
    count--; // Decrement the counter
}

File: src/data/snippets/foreach-loops/snippet.c
#include <stdio.h>

int main() {
    // Initialize an array of numbers
    int numbers[] = {5, 12, 15, 20, 25};
    int length = sizeof(numbers) / sizeof(numbers[0]);

    // Print each number using a for-each loop
    for (int i = 0; i < length; i++) {
        printf("Number: %d\n", numbers[i]);
    }
    
    return 0;
}
